integration-tests/src/test/java/integration/system/bundles/ContentPackStreamsTest.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package integration.system.bundles;
+
+import com.google.common.net.HttpHeaders;
+import com.jayway.restassured.response.ValidatableResponse;
+import integration.BaseRestTest;
+import integration.RequiresAuthentication;
+import integration.RequiresVersion;
+import org.junit.Test;
+
+import java.net.URI;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.notNullValue;
+
+@RequiresVersion(">=1.2.0")
+@RequiresAuthentication
+public class ContentPackStreamsTest extends BaseRestTest {
+
+    URI uri;
+
+    @Test
+    public void createContentPackWithStreams() {
+        final ValidatableResponse validatableResponse = given().when()
+                .body(jsonResourceForMethod()).post("/system/bundles")
+                .then()
+                .statusCode(201)
+                .statusLine(notNullValue());
+
+        uri = URI.create(validatableResponse.extract().header(HttpHeaders.LOCATION));
```

Comment:
JUnit requires test methods to be of type `void` (i. e. have no return value).

The only assertion I could think of would be quite short (e. g. testing the URI's path started with `/system/bundles`). There are implicit assertions here which are already tested, like the response status code (201) and that the status line ("HTTP/1.1 200 **OK**") wasn't blank.